.github/renovate.json5

```
@@ -0,0 +1,54 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended",
```

```suggestion
    "config:best-practices",
```

This adds the following on top of `config:recommended`:

```
{
  "configMigration": true,
  "extends": [
    "config:recommended",
    "docker:pinDigests",
    "helpers:pinGitHubActionDigests",
    ":pinDevDependencies"
  ]
}
```

https://docs.renovatebot.com/presets-config/